elastic/beats
# [metricbeat][system/process, system/process_summary]: mark module as healthy if metrics are partially filled

- Bug

## Proposed commit message

- If metrics are partially available for system.process and system.process_summary, do not mark metricset as degraded.
- Instead, keep it healthy and display the message

## Checklist





## How to test this PR locally



- Clone from this branch
- Do `mage package` from `x-pack/agentbeat`
- Run elastic agent with new `agentbeat` binary
- Run as unprivileged mode, and run `elastic-agent status --output=json`
- You should see `HEALTHY` state and error message displayed

## Related issues

- Closes #40542

## Use cases



## Screenshots

<img width="1467" alt="Screenshot 2024-08-22 at 12 38 05 PM" src="https://github.com/user-attachments/assets/4e0b80c7-4e80-4d80-8426-8fda92a4dfc8">
<img width="1464" alt="Screenshot 2024-08-22 at 12 38 13 PM" src="https://github.com/user-attachments/assets/eb3f6b21-a217-4839-aa18-eeb801cc4198">
